cosmwasm/enclaves/rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2020-10-25
+nightly
```

Comment:
We usually want to pin the enclave's compiler version, because:
- Teaclave only explicitly support specific nightly versions.
- We want to make sure our build are relatively reproducible, and we all see the same compiler bugs if they pop up.
- This also makes sure we all work with the same language features.

Reply:
aha!

this is key info